src/app/filter/examples/filter-save-example.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="padding-15">
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="form-group">
+        <pfng-filter #filter
+                     [config]="filterConfig"
+                     (onChange)="filterChanged($event)"
+                     (onDelete)="deleteSavedFilter($event)"
+                     (onFieldSelect)="filterFieldSelected($event)"
+                     (onSave)="saveFilter($event)"
+                     (onTypeAhead)="filterQueries($event)"></pfng-filter>
+        <div class="padding-top-15">
+          This example lazily loads filters in addition to saving a custom filter.
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <div class="padding-top-15">
+        <button class="btn btn-default primary-action" type="button"
+                (click)="loadSavedFilters()">Load Saved Filters
+        </button>
+      </div>
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Valid Items:</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
```

Comment:
Not sure how this will work, with this being `col-md-` and other pieces being `col-xs-`